pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/PulsarAuthorizationProvider.java

```diff
@@ -263,7 +263,10 @@ public CompletableFuture<Void> revokePermissionAsync(TopicName topicName, String
                         policies.auth_policies.getTopicAuthentication()
                                 .computeIfPresent(topicName.toString(), (k, v) -> {
                                         v.remove(role);
-                                        return null;
+                                        if (v.isEmpty()) {
```

Comment:
I would be helpful to keep the meaningful variable names for `k` and `v` here, like the `PersistentTopicsBase` has.

Reply:
Fixed